appveyor.yml

```diff
@@ -0,0 +1,14 @@
+image: Ubuntu
```

Comment:
We want windows for appveyor.

Reply:
@majurg sure I will be adding the windows build too.